scripts/wtr.js

```diff
@@ -34,6 +34,18 @@ function runTests() {
     if (fs.existsSync(`${itFolder}/${wtrTestsFolderName}`)) {
       console.log(`Installing dependencies in ${itFolder}`);
 
+      // Set up an empty node_modules and package.json before running Flow build
+      // Those will get cleaned up by `flow:build-frontend` unless they existed
+      // before, but we need them for running web-test-runner
+      const nodeModules = `${itFolder}/node_modules`;
+      const packageJson = `${itFolder}/package.json`;
+      if (!fs.existsSync(nodeModules)) {
+        fs.mkdirSync(nodeModules);
+      }
+      if (!fs.existsSync(packageJson)) {
+        fs.writeFileSync(packageJson, '{}');
```

Comment:
Does `flow:build-frontend` update `package.json` with some meaningful content?

Reply:
Yes it does, it adds dependencies needed for the tests, like Polymer and Lit.